Jellyfin.Plugin.Reports/Api/Common/ReportBuilderBase.cs

```diff
@@ -1,4 +1,4 @@
-﻿#nullable disable
+#nullable disable
```

Comment:
if this removes the BOM this is fine, but please remove it, if it adds one (same for all 3 files)

Reply:
Yeah, these changes were removing the UTF8 BOMs